text/0605-eks-rewrite.md

```diff
@@ -0,0 +1,448 @@
+# EKS L2 Re-write
+
+- **Original Author(s)**: @xazhao
+- **Tracking Issue**: [#605](https://github.com/aws/aws-cdk-rfcs/issues/605)
+- **API Bar Raiser**: @iliapolo
+
+The `eks-v2-alpha` module is a rewrite of the existing `aws-eks` module. This
+new iteration leverages native L1 CFN resources, replacing the previous custom
+resource approach for creating EKS clusters and Fargate Profiles. Beyond the
+resource types change, the module introduces several other breaking changes
+designed to better align the L2 construct with its updated implementation. These
+modifications collectively enhance the module's functionality and integration
+within the AWS ecosystem, providing a more robust and streamlined solution for
+managing Elastic Kubernetes Service (EKS) resources.
+
+This RFC primarily addresses the distinctions between the new module and the
+original EKS L2 construct. Comprehensive use cases and examples will be
+available in the README file of the forthcoming `eks-alpha-v2` module.
```

Comment:
eks-v2-alpha or do we want to reference the actual module name aws-eksv2-alpha?

Reply:
`eks-v2-alpha` looks more clean to me. I'm open to either way though.